### DISCUSSION TITLE
Restriction on "Authorize the application with your OneDrive Account"?

### DISCUSSION THREAD

**Discussion by CharlesMAtkinson** [abraunegg/onedrive#2799] [General]
After installing onedrive (v2.4.25-1+np3+5.2):
```
$ onedrive
Configuring Global Azure AD Endpoints
Authorize this app visiting:

https://login.microsoftonline.com/common/oauth2/v2.0/authorize?client_id=d5<redacted>0c&scope=Files.ReadWrite%20Files.ReadWrite.All%20Sites.ReadWrite.All%20offline_access&response_type=code&prompt=login&redirect_uri=https://login.microsoftonline.com/common/oauth2/nativeclient

Enter the response uri:
```
So far so good.  Pasted that link into Firefox.  At the sign in screen, entered my email address.  Clicked Next.  Saw "We couldn't find an account with that username. Try another, or get a new Microsoft account"

The email address is known to work.  Normally after entering that email address, Microsoft responds with: 
```
Enter code
We just sent a code to <my email address>
```
Previously, when working with the same organisation via conventional username and password authentication, onedrive "just worked".  I am now working with another team, hence the change of user.

Is the "We couldn't find an account with that username" caused by the user being set up for authentication by code sent to email address?

Best

Charles

**Comment by abraunegg** [comment]
@CharlesMAtkinson 
> Is the "We couldn't find an account with that username" caused by the user being set up for authentication by code sent to email address?

Unfortunately you need to raise that question with who ever is looking after your organisation Azure / Microsoft 365 platform or Microsoft.

It sounds like the new account | email address is not set up correctly to access Microsoft OneDrive services.

I cannot assist with this query any further than the above - sorry.

**Reply by CharlesMAtkinson** [reply]
Thanks abrounegg, that was very quick and is clear
